README.md

```diff
@@ -0,0 +1,7 @@
+# ESP-BeagleboneHomeAutomation
+This is a kind of home automation system using this two technologies (ESP32 and Beaglebone Black Wireless) 
+that establish communication by bluetooth low energy.
+Inside of BBB is running a fuzzy-logic system that takes control of the lights and air-conditioned behavior
```

Comment:
Here you can add the libraries that you are using, scikit-fuzzy and bluepy, and its owners